### ISSUE TITLE
Bug when training with different resolutions

### ISSUE THREAD

**Issue by MaverickRen** [NVIDIA/Cosmos#148]
The default resolution for training the Cosmos autoregressive model is 640x1024. I changed it to 320x512, but I found that the model trained with this resolution results in nan logits during inference. Is there something in the model's design related to resolution that might be causing this?

**Comment by mingyuliutw** [comment]
@MaverickRen  I think you are already using. Look forward to your new issues in those repos.

https://github.com/nvidia-cosmos/cosmos-predict1
https://github.com/nvidia-cosmos/cosmos-transfer1
